osbuild/__init__.py

```diff
@@ -116,13 +123,25 @@ def run(self, argv, binds=None, readonly_binds=None, **kwargs):
             f"--machine={self.machine_name}",
             f"--directory={self.root}",
             f"--bind={libdir}/osbuild-run:/run/osbuild/osbuild-run",
-            *[f"--bind={b}" for b in (binds or [])],
+            f"--bind-ro={libdir}/osbuild:/run/osbuild/osbuild",
+            *[f"--bind-ro={b}" for b in [self.api + ":" + "/run/osbuild/api",
+                                         *(binds or [])]],
```

Comment:
This should probably go into the `--bind-ro` line below, keeping `--bind` for `binds`.

Reply:
Indeed. Fixed.